.github/workflows/workflow-installer.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: project auto-detection
         id: project-auto-detection
-        if: inputs.auto-detect == 'true'
+        if: ${{ inputs.auto-detect == 'true' }}
```

Comment:
personally, I would not go with the variable definition where it is not really needed... its a lot personal preference, so if you prefer the new syntax, I am also fine with it :)

Reply:
no.. i think the other way is just inputs.auto-detect == 'true' (as you had it before).
For an if parts I do not think there is any difference in semantics, just in syntax :-D